src/vision/camera_debug.py

```diff
@@ -0,0 +1,8 @@
+import visionfunctions as vision
+
+vision.camera_init()
```

Comment:
Debug programs?

Reply:
All of thses py files are quinn things- they kind of just found their way into the pull request